client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -28,36 +28,10 @@
 	}
 }
 
-.domain-management-list__notice {
-	margin-bottom: 0;
-}
-
-.domain-management-list-item {
-	&.busy {
```

Comment:
This `.domain-management-list-item` class with a `.busy` modifier is not used anymore